### PR TITLE
Add new Control system overview image

### PR DESCRIPTION
**Pull request by yellowjaguar5**
This PR was branched off of #200 per request by @AustinShalit and @Daltz333. The image added was designed by Stefen Acepcion.

### CODE REVIEW

**Comment by stefacep** [comment]
I approve the use of the image.

**Comment by yellowjaguar5** [comment]
@AustinShalit can you rebuild and review?

**Comment by yellowjaguar5** [comment]
Depends on #610

**Comment by yellowjaguar5** [comment]
@Daltz333 Could you rebuild?

**Comment by Daltz333** [comment]
Per discussion with Austin. 
image

File: source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst [hunk 1]
```diff
@@ -10,7 +10,9 @@ The goal of this document is to provide a brief overview of the hardware compone
 Overview of Control System
 -------------------------------
 
-.. image:: images/control-system-hardware/FRC-Control-System-Layout.svg
+.. figure:: images/control-system-hardware/FRC-Control-System-Layout.svg
+
+  Diagram countacy of FRC Team 3161 and Stefen Acepcion.
```
Comment (Daltz333) [inline source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst:15]: Indentation looks wrong here.

Comment (yellowjaguar5) [reply source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst]: It looks like it builds properly, what do you mean?

Comment (Daltz333) [reply source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst]: Indentation should be 3 spaces not 2. It should align with the first character of ``figure``

Comment (yellowjaguar5) [reply source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst]: Huh, I added 3 spaces to the indent in 2db721e, but the build failed with the error:
`/home/vsts/work/1/s/source/docs/getting-started/getting-started-frc-control-system/how-to-wire-a-robot.rst:13: WARNING: Figure caption must be a paragraph or empty comment.
`
- Interesting that there are also 3 spaces in control-system-hardware, line 15, but no error was thrown.
- Builds passed in 09c9a8b and locally with 2 spaces for indents, so the third space seems to be the cause of the build error.

**Pull request merged**